tests/validation/tests/v3_api/resource/terraform/rke2/master/instances_server.tf

```diff
@@ -262,7 +262,7 @@ resource "aws_lb_listener" "aws_nlb_listener_443" {
 
 resource "aws_route53_record" "aws_route53" {
   zone_id            = "${data.aws_route53_zone.selected.zone_id}"
-  name               = "${var.resource_name}-route53"
+  name               = "${var.resource_name}"
```

Comment:
Would suggest to keep the resource name with route53 suffix to keep it consistent with other resource naming. Delete job may be dependent on this.

Reply:
k3s doesn't have this suffix on its route53 creation and it seems redundant to me to have it